lib/services.js

```diff
@@ -478,6 +479,7 @@ export default {
         const mpuBucketName = `${constants.mpuBucketPrefix}${bucketName}`;
         metadata.getBucket(mpuBucketName, log, (err, mpuBucket) => {
             if (err) {
+                log.error('error from metadata', { error: err });
```

Comment:
Given our current... difficulties.. at investigating issues, I do not believe reducing the number of error logs in the lower layers to be a very good choice. Most errors will be always interpreted as errors, except very specific cases that we should Know and control.

These specific cases, I'm back to the 404 on bucket get, will have various interpretations depending on the context of the request. Those should be logged in the upper API layer. But most of the errors we encounter do not have this ambiguity, and should always be logged.

Worded another way, errors should always be logged at the very moment where the error is understood as a `500 internal error` issue. Which means a 404 within a validateMultipart should indeed log error. but a 500 in any context should always log error too.

That's my (and not only mine to be honest) point and hope for this PR which might lead to a slightly wider scope than probably originally thought.